src/app/portal/features/site-header/site-header.component.html

```diff
@@ -17,13 +17,13 @@
 
     <mat-nav-list class="my-8 basis-1/2 text-center">
       <a
-        [id]="route === '/' ? 'activePage' : ''"
+        [id]="route() === '' ? 'activePage' : ''"
         mat-button
         routerLink="/"
         class="homeLink mx-1"
         >Home</a
       ><a
-        [id]="route.startsWith('/browse') ? 'activePage' : ''"
+        [id]="route() === 'browse' ? 'activePage' : ''"
```

Comment:
This looks a little odd to me, but it's personal opinion. It is indeed technically not Browse Data.

Reply:
What do you think looks odd? Should I rename it to `baseRoute`?

Reply:
That it does not light up when you are in the dataset details (SDSV) page.

Reply:
Ah ok, but that's not something that changed in this PR. It's because the datasets are now shown under /dataset.

I changed this in f510c47. Now it also lights up when you view a specific dataset.